src/BaGet.Core/Search/DatabaseSearchService.cs

```diff
@@ -125,7 +125,12 @@ public async Task<DependentsResponse> FindDependentsAsync(string packageId, Canc
                 .OrderByDescending(p => p.Downloads)
                 .Where(p => p.Dependencies.Any(d => d.Id == packageId))
                 .Take(20)
-                .Select(p => p.Id)
+                .Select(r => new DependentResult()
```

Comment:
Minor style nitpick:

```suggestion
                .Select(r => new DependentResult
```